packages/libs/deer/README.md

```diff
@@ -0,0 +1,86 @@
+# Deer
+
+`deer` is a backend agnostic deserialization framework, featuring meaningful error
+messages and context (utilizing [`error-stack`](https://crates.io/crates/error-stack)) and
+a fail-slow behavior by default.
+
+## Fail-Slow
+
+Currently, Rust deserializers are good at one thing: parsing a lot of data correct and
```

Comment:
It occurs to me that this might come across as unnecessarily harsh. One might believe that Rust deserializers are good at more than _just_ one thing.

We should still make the case that most Rust deserializers have been developed with correctness and speed in mind, but can avoid saying they are "good at one thing", as this runs the risk of unintentionally sounding exclusive/exhaustive.

Reply:
> It occurs to me that this might come across as unnecessarily harsh.

I guess that's where my inner German surfaced^^

I changed the wording in [2bb0b1b](https://github.com/hashintel/hash/pull/1163/commits/2bb0b1b818abf9c9c3622249777410805a730abc), is this better?